ocs_ci/ocs/perftests.py

```diff
@@ -1023,3 +1027,114 @@ def cleanup_testing_pod(self):
         except Exception:
             log.warning("The POD failed to delete")
             pass
+
+    def get_performance_metrics(self, start_time):
```

Comment:
Can we also have an option to run this on a selected pod instead of all pods? 
it provides the ability to choose the metric one likes to collect as opposed to all by default. Maybe pod can also take `all` in which case it could collect metrics from all the pods of a namespace.  
```
def get_performance_metrics(self, start_time, pod):
```
(It may be a good idea to have the namespace included as an option too, that can go in a subsequent PR)

Reply:
Sure. I will make changes to run for specific pod or all pods
Also i will add "step" parameter as an option too, so that tests that take longer time will not result in data flooding
i.e. if test completes in 2 mins we can have step size as 10seconds
Whereas if test takes few hours we can provide step size as 10 mins